hw-event-proxy/cmd/main.go

```diff
@@ -201,8 +205,9 @@ func handleHwEvent(bodyBytes []byte) {
 	e.SetData(data)
 	err = publishHwEvent(e)
 	if err != nil {
-		log.Errorf("failed to publish hw event: %v", err)
+		return fmt.Errorf("failed to publish hw event: %v", err)
 	}
+	return nil
```

Comment:
not sure we should return nil here.. may be we should return a general error?

Reply:
`nil` means no error here.